### ISSUE TITLE
Flags do not appear in mobile web browser

### ISSUE THREAD

**Issue by esmlima** [cedvdb/flutter_circle_flags#9]
I don't understand why, but I'm facing different behaviour in flutter web projects when accessed from Desktop vs Mobile.

My demo project:
https://payment.gula.mobi/#/

On desktop it works fine:
![image](https://user-images.githubusercontent.com/110770797/210071002-517ee60d-2d79-4348-934a-edf77b5baff7.png)

On Android mobile, using Chrome:
![Mobile Chrome](https://user-images.githubusercontent.com/110770797/210071183-e8494d15-41e2-451a-b414-1d306e33db66.jpeg)

On  Android mobile, using Firefox:
![Mobile Firefox](https://user-images.githubusercontent.com/110770797/210071221-a0cad9bb-0901-4b77-9ac8-01d84c7f9d08.jpeg)

On iPhone, using Safari:
![iPhone Safari](https://user-images.githubusercontent.com/110770797/210072865-fdb3311d-1bbc-43cf-9e79-aa1917ad59d5.jpeg)

**Comment by cedvdb** [comment]
Good catch, hopefully someone can look into it. It might be due to svgs.

**Comment by arjittw** [comment]
Actually, they are not working in html renderer, only working in CanvasKit renderer. By default, flutter uses CanvasKit on desktop browsers and html renderer on mobile browsers.

**Comment by cedvdb** [comment]
This is probably due to the move to flutter-svg, as I recall it was working previously

**Comment by blopker** [comment]
Related: https://github.com/dnfield/flutter_svg/issues/845

Yeah, looks like it's an issue with flutter_svg.

**Comment by blopker** [comment]
Made a PR, hope it helps: https://github.com/cedvdb/flutter_circle_flags/pull/11

**Comment by cedvdb** [comment]
Thanks a lot @blopker  I'll get on it this weekend, I'm drowning in high priority tasks right now

**Comment by cedvdb** [comment]
should be fixed in 1.0.4

**Comment by agordeev** [comment]
This issue happens again in 5.0.0

**Comment by cedvdb** [comment]
@cyrilmata the original fix is linked above and can be reimplemented. The process has to be repeated each time the flags are regenerated though, which is annoying. 

This issue in the original repository would fix it at the source: https://github.com/HatScripts/circle-flags/issues/105
